app/src/main/java/com/nutomic/syncthingandroid/service/EventProcessor.java

```diff
@@ -167,11 +188,29 @@ public void onEvent(Event event) {
                             new String[]{updatedFile.getPath()});
                 }
                 break;
+            case "ItemStarted":
+                break;
+            case "LocalIndexUpdated":
+                // ToDo
+                Log.v(TAG, "Unhandled event " + event.type);
+                break;
+            case "LoginAttempt":
+                break;
             case "Ping":
                 // Ignored.
                 break;
-            default:
+            case "RemoteIndexUpdated":
+                // ToDo
                 Log.v(TAG, "Unhandled event " + event.type);
+                break;
+            case "Starting":
+                break;
+            case "StartupComplete":
+                break;
+            case "StateChanged":
+                break;
```

Comment:
You don't need to explicitly break on every event.
Just let them fall through and break before default.

Also, no point logging stuff about indexUpdate et all events if they are knowingly not handled. What value does logging stuff you know you don't handle provide?